tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -147,22 +164,16 @@ def createProxy(self, userName, group, time, rfc=True, limit=False, vo=None, rol
         if 'bad decrypt' in retVal['Message']:
           return S_ERROR("Bad passphrase")
         return S_ERROR("Can't load %s" % userKeyFile)
-      result = chain.generateProxyToFile(self.proxyPath, time * 3600,
-                                         limited=limit, diracGroup=group,
-                                         rfc=rfc)
+      result = chain.generateProxyToFile(self.proxyPath, time * 3600, diracGroup=group)
       if not result['OK']:
         return result
     else:
       cmd = 'voms-proxy-fake --cert %s --key %s -q' % (userCertFile, userKeyFile)
       cmd += ' -hostcert %s -hostkey %s' % (self.hostCert, self.hostKey)
       cmd += ' -uri fakeserver.cern.ch:15000'
-      cmd += ' -voms "%s%s"' % (vo, role and ':%s' % role or '')
+      cmd += ' -voms "%s"' % vo
```

Comment:
@chaen will fix `ValueError: too many values to unpack`, removed `:` symbol that using for parsing in `decodeVOMSExtension`

Reply:
ok thanks